tests/unit/clients/python/test_io.py

```diff
@@ -73,12 +73,40 @@ def test_input_lines_with_empty_filepath_and_lines():
         for _ in lines:
             pass
 
+@pytest.mark.parametrize(
+    'size, sampling_rate',
+    [
+        (None, None),
+        (1, None),
+        (None, 0.5),
+    ],
+)
+def test_input_lines_with_jsonlines_file(size, sampling_rate):
+    result = list(from_lines(filepath='tests/unit/clients/python/docs.jsonlines', size=size, sampling_rate=sampling_rate))
```

Comment:
also please use relative paths to a `cur_dir` (the directory of the current source code file). This makes it easier to run tests independently from the directory where this is run from.